docs/reference/cli.md

```diff
@@ -204,9 +191,9 @@ quickwit index search
 
 *Options*
 
-`--index` id of the target index.    
+`--index` ID of the target index.    
 `--config` Quickwit config file.    
-`--query` Query expressed in Tantivy syntax.    
+`--query` Query expressed in natural query language (barack AND obama) OR "president of united states").    
```

Comment:
I think we need a link here or reference tantivy syntax, or we let the user guess the actual syntax

Reply:
I put a link to the query language in the description.

Reply:
I think it would be good to have it also in the cli for users that explore the cli without the docs

Reply:
Will do.